tests/hardhat/PoolRegistry.ts

```diff
@@ -335,14 +335,24 @@ describe("PoolRegistry: Tests", function () {
   });
 
   it("Metadata", async function () {
-    await poolRegistry.updatePoolMetadata(0, {
-      riskRating: 2,
-      category: "Hign market cap",
-      logoURL: "http://venis.io/pool1",
-      description: "An sample description",
-    });
+    const riskRating = 2;
+    const category = "Hign market cap";
+    const logoURL = "http://venis.io/pool1";
```

Comment:
Definitely need to change one letter here, not sure which one though 😂

Reply:
done